monai/transforms.py

```diff
@@ -0,0 +1,62 @@
+            SpatialPadd(keys=["image", lbl_key], spatial_size=(64, 128, 128), method="symmetric"),
+            # SpatialPadd(keys=["image", lbl_key], spatial_size=(123, 255, 214), method="symmetric"),
+            # RandSpatialCropSamplesd(keys=["image", lbl_key], roi_size=crop_size, num_samples=num_samples_pv, random_center=True, random_size=False),
+            RandCropByPosNegLabeld(keys=["image", "label"], label_key="label",
+                                   spatial_size=crop_size, pos=1, neg=1, num_samples=num_samples_pv, 
+                                   # if num_samples=4, then 4 samples/image are randomly generated
+                                   image_key="image", image_threshold=0.),
+            Rand3DElasticd(keys=["image", lbl_key], sigma_range=(3.5, 5.5), magnitude_range=(25, 35), prob=0.5),
+            RandBiasFieldd(keys=["image", lbl_key], coeff_range=(0.0, 0.5), prob=0.25, degree=3),
+            RandFlipd(keys=["image", lbl_key], spatial_axis=[0], prob=0.50,),
+            RandFlipd(keys=["image", lbl_key], spatial_axis=[1], prob=0.50,),
+            RandFlipd(keys=["image", lbl_key],spatial_axis=[2],prob=0.50,),
+            # RandRotate90d(keys=["image", lbl_key], prob=0.10, max_k=3,),
+            Orientationd(keys=["image", lbl_key], axcodes="RPI"),   # NOTE: if not using it here, then it results in collation error
+            # HistogramNormalized(keys=["image"], mask=None),
+            NormalizeIntensityd(keys=["image"], nonzero=False, channel_wise=True),
+            # ToTensord(keys=["image", lbl_key]), 
+        ])
+
+def val_transforms(lbl_key="label"):
+    return Compose([
+            LoadImaged(keys=["image", lbl_key]),
+            EnsureChannelFirstd(keys=["image", lbl_key]),
+            Orientationd(keys=["image", lbl_key], axcodes="RPI"),
+            Spacingd(keys=["image", "label"], pixdim=(1.0, 1.0, 1.0), mode=("bilinear", "bilinear"),),
+            CropForegroundd(keys=["image", lbl_key], source_key="image"),
+            # SpatialPadd(keys=["image", lbl_key], spatial_size=(123, 255, 214), method="symmetric"),
+            # HistogramNormalized(keys=["image"], mask=None),
+            NormalizeIntensityd(keys=["image"], nonzero=False, channel_wise=True),
+            # ToTensord(keys=["image", lbl_key]),
+        ])
+
+def test_transforms(lbl_key="label"):
+    return Compose([
+            LoadImaged(keys=["image", lbl_key]),
+            EnsureChannelFirstd(keys=["image", lbl_key]),
+            Orientationd(keys=["image", lbl_key], axcodes="RPI"),
+            Spacingd(keys=["image", "label"], pixdim=(1.0, 1.0, 1.0), mode=("bilinear", "bilinear"),),
+            CropForegroundd(keys=["image", lbl_key], source_key="image"),
+            # AddChanneld(keys=["image", lbl_key]),
+            # HistogramNormalized(keys=["image"], mask=None),
+            NormalizeIntensityd(keys=["image"], nonzero=False, channel_wise=True),
+            # ToTensord(keys=["image", lbl_key]),
```

Comment:
you might want to minimize code duplication to avoid augmentation to be different between the train/eval/test

Reply:
This has been fixed in 407ae043c35b88eccb3f73678b9cb0323b23df3f